lib/js/components/DatePickers/DatePicker/DatePicker.vue

```diff
@@ -0,0 +1,278 @@
+<template>
+	<div
```

Comment:
on figma the entire component is as _trigger, along with the label - should we listening a toggle event on the entire component?

Reply:
What you mean exactly? That we should listen on root component for the click event to trigger picker?

Reply:
yes

Reply:
I am not convinced since we would do it right now. Further more, we would toggle picker when clicking on error message. I think the wrapper got this name for convenience and girls didn't think it thoroughly.